src/Projectionist.php

```diff
@@ -53,6 +53,13 @@ public function addProjector($projector): Projectionist
         return $this;
     }
 
+    public function allEventHandlers(): EventHandlerCollection
+    {
+        $eventHandlers = $this->projectors->all()->merge($this->reactors->all())->toArray();
```

Comment:
Why first calling `toArray` and then wrap it in a new `EventHandlerCollection` ? 

```
$this->projectors->all()->merge($this->reactors->all())
```

This already is an `EventHandlerCollection`

Reply:
Currently `EventHandlerCollection` does not extend from `Collection` and has no `merge` function.

Hold my Gini while I refactor some more.